src/coreclr/debug/createdump/crashinfounix.cpp

```diff
@@ -262,7 +262,7 @@ CrashInfo::GetDSOInfo()
 void
 CrashInfo::VisitModule(uint64_t baseAddress, std::string& moduleName)
 {
-    if (baseAddress == 0 || baseAddress == m_auxvValues[AT_SYSINFO_EHDR] || baseAddress == m_auxvValues[AT_BASE]) {
+    if (baseAddress == 0 || baseAddress == m_auxvValues[AT_SYSINFO_EHDR]) {
```

Comment:
This was needed to fix createdump on linux-alpine. see https://github.com/dotnet/runtime/pull/42565/files

Reply:
Yes. It is ok now for alpine because the stack unwinding has a check to prevent the uncontrolled looping.

Reply:
Errors should always print something out I think and TRACE doesn't (only --diag).